autopep3

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+SCRIPT_DIR="$PWD/$(dirname $0)"
+
+export PYTHONPATH="$SCRIPT_DIR/.pip3/lib/python3.5/site-packages"
+export PATH="$SCRIPT_DIR/.pip3/bin:$PATH"
+
+if [ ! autopep8 > /dev/null 2>/dev/null ] ;
```

Comment:
This square brackets(`[]`) are not required. This condition with square brackets is always false either if `autopep8` is installed or not installed.

BTW I suppose `venv` is useful in such case. Sample is  https://gist.github.com/syohex/44f551b380f661cdceb0ed7e2ae34fec. `venv` is standard library since python 3.3 however ubuntu python does not bundle it same as `pip` and we need to install it by `apt install python3-venv`.

Reply:
Thanks. Shell scripts are too difficult 😭